General/Glossary.md

```diff
@@ -52,6 +52,8 @@ Cog is the name of the [virtual machine](#virtual-machine) used by Pharo.
 It is also used by other programming languages such as [Squeak](#squeak), Newspeak and Cuis.
 
 ## Context
+A *Context* represent a program execution. It will store for example the `CompiledMethod` currently been executed, the receiver and arguments of the message that invoked this `CompiledMethod`, the temporary variables, the stack of calls until this moment of the execution, ...
```

Comment:
```suggestion
A *Context* represent a program execution. It will store for example the `CompiledMethod` being currently executed, the receiver and arguments of the message that invoked this `CompiledMethod`, the temporary variables, the stack of calls until this moment of the execution, ...
```